admin-dev/themes/new-theme/js/pages/manufacturer/index.js

```diff
@@ -23,19 +23,31 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
-import Grid from '../../../components/grid/grid';
-import SortingExtension from '../../../components/grid/extension/sorting-extension';
-import FiltersResetExtension from '../../../components/grid/extension/filters-reset-extension';
-import ReloadListActionExtension from '../../../components/grid/extension/reload-list-extension';
-import ColumnTogglingExtension from '../../../components/grid/extension/column-toggling-extension';
-import SubmitRowActionExtension from '../../../components/grid/extension/action/row/submit-row-action-extension';
-import SubmitBulkExtension from '../../../components/grid/extension/submit-bulk-action-extension';
-import BulkActionCheckboxExtension from '../../../components/grid/extension/bulk-action-checkbox-extension';
-import ExportToSqlManagerExtension from '../../../components/grid/extension/export-to-sql-manager-extension';
+import FormSubmitButton from '../../components/form-submit-button';
+import Grid from '../../components/grid/grid';
+import SortingExtension from '../../components/grid/extension/sorting-extension';
+import FiltersResetExtension from '../../components/grid/extension/filters-reset-extension';
+import ReloadListActionExtension from '../../components/grid/extension/reload-list-extension';
+import ColumnTogglingExtension from '../../components/grid/extension/column-toggling-extension';
+import SubmitRowActionExtension from '../../components/grid/extension/action/row/submit-row-action-extension';
+import SubmitBulkExtension from '../../components/grid/extension/submit-bulk-action-extension';
+import BulkActionCheckboxExtension from '../../components/grid/extension/bulk-action-checkbox-extension';
+import ExportToSqlManagerExtension from '../../components/grid/extension/export-to-sql-manager-extension';
 
 const $ = window.$;
 
 $(() => {
+  const addressGrid = new Grid('manufacturer_address');
```

Comment:
since the grids are init the same way you could use a loop like:

```js
['manufacturer_address', 'manufacturer'].forEach(function initGrid(gridId) {
    const grid = new Grid(gridId);
    ...
});
```

Reply:
This is fixed in another PR, so I'll either rebase or it will be rebased in that PR.

Reply:
Yes we need to be careful to merge the other one first, and then rebase this one

Reply:
Actually it doesn't matter if we rebase this one or the other ^^